aws/rust-runtime/aws-credential-types/src/cache/expiring_cache.rs

```diff
@@ -59,6 +59,14 @@ where
             .map(|(creds, _expiry)| creds)
     }
 
+    /// Attempts to load the cached value if it has been set
+    ///
+    /// # Panics
+    /// This function panics if it is called from an asynchronous context
+    pub fn try_blocking_get(&self) -> Option<T> {
```

Comment:
Doesn't look like this is used?

Reply:
yeah deleted, an artifact from a previous impl